pom.xml

```diff
@@ -55,4 +55,24 @@
       <url>http://repo.jenkins-ci.org/public/</url>
     </pluginRepository>
   </pluginRepositories>
+
+  <dependencies>
+    
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.12</version>
+      <scope>test</scope>
+    </dependency>

```

Comment:
(Note for me)
junit-4.11 will be used without this.
(I'm not sure who defines that version)